docs/admin/authentication.md

```diff
@@ -486,6 +486,21 @@ An unsuccessful request would return:
 
 HTTP status codes can be used to supply additional error context.
 
+#### Using OpenStack Keystone authentication Webhook
+
+There are two parts to authentication with OpenStack Keystone. On the client-side, you can use `--auth-provider=openstack` option to reuse the `OS_*` environment variables:
+```
+kubectl config set-credentials openstackuser --auth-provider=openstack
+```
+Note that you will also need to use `kubectl config set-context` and `kubectl config use-context` to activate the context. Please make sure you have `OS_DOMAIN_NAME` to use Keystone V3 API.
```

Comment:
Adds an auth config file example to illustrate #60200?

Reply:
Good information, but its unclear on how someone would actually use these commands and in what order to authenticate to the cluster. I'd recommend showing all three commands in sequence, explaining what they do to provide the authentication, and prefixing the whole thing by asserting that the administrator specifically needs to have OS_DOMAIN_NAME set. There's an implication of other OS_* environment variables that may need to be set - I'm not sure what those may be, but if any are critical, please include them in a "to use this, you need..." sort of requirements note above the sequence of commands.